docs/docusaurus/static/_redirects

```diff
@@ -73,7 +73,7 @@ http://legacy.016.docs.greatexpectations.io/* http://docs.greatexpectations.io 3
 /en/latest/guides/how_to_guides/miscellaneous/how_to_template.html https://legacy.docs.greatexpectations.io/en/latest/guides/how_to_guides/miscellaneous/how_to_template.html
 /en/latest/guides/how_to_guides/miscellaneous/how_to_add_and_test_a_new_sqlalchemydataset_class.html https://legacy.docs.greatexpectations.io/en/latest/guides/how_to_guides/miscellaneous/how_to_add_and_test_a_new_sqlalchemydataset_class.html
 /en/latest/guides/how_to_guides/miscellaneous/how_to_use_official_docker_images.html https://legacy.docs.greatexpectations.io/en/latest/guides/how_to_guides/miscellaneous/how_to_use_official_docker_images.html
-/en/latest/guides/how_to_guides/miscellaneous/how_to_setup_opsgenie_alert_notifications.html https://legacy.docs.greatexpectations.io/en/latest/guides/how_to_guides/miscellaneous/how_to_setup_opsgenie_alert_notifications.html
+/en/latest/guides/how_to_guides/miscellaneous/how_to_setup_opsgenie_alert_notifications.html /docs/0.18/reference/api/checkpoint/opsgeniealertaction_class/
```

Comment:
Why aren't we completely removing opgenie here if it is not supported? Is it because the code still exists? When we remove that code, will we have to remove this code and update this file again?

Reply:
That is, if we need to remove it later, could you add this to the ticket you filed, CORE-600?

Reply:
hmmm good point! I forgot this part of our support posture -https://docs.greatexpectations.io/docs/application_integration_support#gx-versions - I'll revise this redirect